Blizzard.IO.RabbitMQ/Rpc/ConcreteMessageSerializationStrategy.cs

```diff
@@ -0,0 +1,26 @@
+﻿using EasyNetQ;
+using System;
+using ISerializer = Blizzard.IO.Core.Rpc.ISerializer;
+
+namespace Blizzard.IO.RabbitMQ.Rpc
+{
+    public class ConcreteMessageSerializationStrategy : IMessageSerializationStrategy
+    {
+        private readonly ISerializer _serializer;
+
+        public ConcreteMessageSerializationStrategy(ISerializer serializer)
+        {
+            _serializer = serializer;
+        }
+
+        public IMessage DeserializeMessage(MessageProperties properties, byte[] body)
```

Comment:
I think that we should leave a comment on this method.
The comment will briefly describe the way we chose to implement the method, and how `RpcClient` should use it from above. For the great good of the next generation obviously...

Reply:
f